x-pack/metricbeat/module/containerd/_meta/config.yml

```diff
@@ -0,0 +1,7 @@
+- module: containerd
+  metricsets: ["cpu", "memory", "blkio"]
+  enabled: false
+  period: 10s
+  hosts: ["localhost:1338"]
+  calcpct: true
```

Comment:
Just a thought around this. Would it make sense to make it available per metricset instead of having it horizontal effect? Sth like:

```
calcpct.cpu: true
calcpct.memory: false
calcpctblkio: true
```

Reply:
For blkio it is not needed(we don't calculate any percentage) but for cpu and memory makes sense

Reply:
5be9960ff9482e9b4b736f3fd742817293cf7c04